docs/user/gcp/customization.md

```diff
@@ -8,6 +8,7 @@ Beyond the [platform-agnostic `install-config.yaml` properties](../customization
 * `controlPlaneSubnet` (optional string): The name of an existing GCP subnet which should be used by the cluster control plane.
 * `computeSubnet` (optional string): The name of an existing GCP subnet which should be used by the cluster nodes.
 * `defaultMachinePlatform` (optional object): Default [GCP-specific machine pool properties](#machine-pools) which apply to [machine pools](../customization.md#machine-pools) that do not define their own GCP-specific properties.
+* `licenses` (optional list of strings): A list of license URLs that should be applied to the compute images. The use of this property in combination with OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE is forbidden. Also, note that use of these URLs will force the installer to copy the source image before being used.
```

Comment:
this needs to include docs from gcp that show where to get these licenses..

Reply:
OK